R-package/tests/testthat/test_lgb.convert_with_rules.R

```diff
@@ -111,13 +111,13 @@ test_that("lgb.prepare_rules2() should work correctly for a dataset with numeric
         expect_identical(rules[["character_col"]], c("a" = 1L, "b" = 2L, "c" = 3L))
         expect_identical(rules[["factor_col"]], c("n" = 1L, "y" = 2L))
 
-        # today, lgb.prepare2() does  not convert numeric  columns
+        # today, lgb.convert() does  not convert numeric  columns
```

Comment:
`convert_with_rules`?

Reply:
```suggestion
        # today, lgb.convert_with_rules() does  not convert numeric  columns
```

Reply:
weirdly GitHub wouldn't let me commit this suggestion, not sure why. So I just pushed a commit: https://github.com/microsoft/LightGBM/pull/3095/commits/0ac6a59189faa327223ed5601ae5dc1fa29bf525